android/src/main/java/com/rnldk/RnLdkModule.kt

```diff
@@ -244,17 +268,19 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
     val newChannelConfig = ChannelConfig.with_default()
     newChannelConfig.set_forwarding_fee_proportional_millionths(10000);
     newChannelConfig.set_forwarding_fee_base_msat(1000);
-    newChannelConfig.set_announced_channel(false); // new channels are private
 
     val handshake = ChannelHandshakeConfig.with_default();
     handshake.set_minimum_depth(1);
-    uc.set_own_channel_config(handshake);
+    handshake.set_announced_channel(false);
+    uc.set_channel_handshake_config(handshake);
+    uc.set_accept_inbound_channels(true);
 
-    uc.set_channel_options(newChannelConfig);
+    uc.set_channel_config(newChannelConfig);
     val newLim = ChannelHandshakeLimits.with_default()
     newLim.set_force_announced_channel_preference(true) // new channels are private
+    newLim.set_trust_own_funding_0conf(true);
 
-    uc.set_peer_channel_config_limits(newLim)
+    uc.set_channel_handshake_limits(newLim);
```

Comment:
@TheBlueMatt  i have some doubts with all that config biz. can you pls take a look and confirm that its all correct?